invisible_cities/cities/sophronia_test.py

```diff
@@ -105,3 +109,7 @@ def test_sophronia_filters_events_with_only_nn_hits(config_tmpdir, sophronia_con
         # (event_number, passed_flag)
         assert     output_file.root.Filters.s12_selector[0][1]
         assert not output_file.root.Filters.valid_hit   [0][1]
+
+        # Check that it's present in /Run/events, but not in /RECO/Events
+        assert event_number == output_file.root.Run.events[0][0]
+        assert event_number not in output_file.root.RECO.Events.col("event")
```

Comment:
This addition to `test_sophronia_filters_events_with_only_nn_hits()` covers the case we're considering (hitless events retaining `Run/events` information but not `RECO/Events` information), but the test itself isn't descriptive of this condition we're checking outside of the comment L113. 

Is it worth separating this case we're checking into a separate test to make clear that this is a condition we're checking for? Or perhaps improving the docstring/name of the test to ensure that this is clear?